playground/index.ts

```diff
@@ -1,3 +1,4 @@
+import 'promise-polyfill/src/polyfill'
```

Comment:
Have you considered using babel and core-js just to not track manually which polyfills we may need? 
Probably overkill and we fine with just js promise but interested in your reasoning.

Reply:
The documentation says that the library requires a single polyfill to work: a Promise polyfill. Having this single polyfill added manually to the playground is a way to check that nothing else is required.

I used Babel with an [automatic core-js polyfilling](https://babeljs.io/docs/en/babel-preset-env#usebuiltins-usage) in a project. Babel added many excess polyfills, the bundle size was too big. It's not a problem on the playground, I just want to mention.